build.sbt

```diff
@@ -50,7 +50,8 @@ libraryDependencies ++= Seq(
   "org.apache.poi"                   % "poi-ooxml"               % "5.2.2",
   "com.github.jsqlparser"            % "jsqlparser"              % "4.6",
   "org.apache.maven"                 % "maven-model"             % "3.9.0",
-  "net.sourceforge.htmlunit"         % "htmlunit"                % "2.70.0"
+  "net.sourceforge.htmlunit"         % "htmlunit"                % "2.70.0",
+  "org.json4s"                      %% "json4s-native"           % "4.0.6"
```

Comment:
We already have a library `io.circe` which is used in propertyFilePass can that be not used for reading json?